docs/structure.rst

```diff
@@ -45,40 +45,35 @@ Overview of directories and files
 
 * ``src/core/``: core of Lambdapi
 
-  * scoping:
-
-      * ``env.ml``: maps identifier -> variable and type
-      * ``pratt.ml``: parsing of applications wrt symbol notations
-      * ``scope.ml``: convert the abstract syntax into terms
-
   * terms:
 
-      * ``term.ml``: internal representation of terms
-      * ``libTerm.ml``: basic operations on terms
-      * ``print.ml``: pretty printing of terms
-
+    * ``term.ml``: internal representation of terms
+    * ``libTerm.ml``: basic operations on terms
+    * ``print.ml``: pretty printing of terms
+    * ``env.ml``: maps identifier -> variable and type
```

Comment:
Remark: env.ml was initially intended for scoping only but it is now used in many core files too. It could be the case that, in core files, it could be replaced by a slightly different data structure.